src/components/aeIdentity/aeIdentity.scss

```diff
@@ -33,35 +19,37 @@
   }
 
   .flex-row{
-    position:relative;
     display: flex;
     justify-content: space-between;
   }
 
   .chunked-address {
-    display: flex;
-    flex-wrap: wrap;
     margin-top: 10px;
-    margin-left: -10px;
   }
 
   .chunked-address .chunk {
```

Comment:
Why don't you just write .chunk? The parent element is not required to select these elements.